modules/SofaBoundaryCondition/CMakeLists.txt

```diff
@@ -1,131 +1,146 @@
 cmake_minimum_required(VERSION 3.12)
-project(SofaBoundaryCondition)
+project(SofaBoundaryCondition VERSION 1.0 LANGUAGES CXX)
```

Comment:
Do we want modules to be independently versioned instead of following SOFA versions?
If no version is given, Sofa_VERSION is set by sofa_auto_set_target_properties

Reply:
1- Dont know for the version (something to discuss I suppose)
2- anyway, if I dont put a version there is an error while doing cmake's config:
```
CMake Error at C:/Program Files/CMake/share/cmake-3.18/Modules/WriteBasicConfigVersionFile.cmake:43 (message):
  No VERSION specified for WRITE_BASIC_CONFIG_VERSION_FILE()
```

Reply:
--> fixed in #1570